Pages/Shared/_Layout.cshtml

```diff
@@ -61,16 +61,21 @@
         }
 
         footer {
-            background-color:powderblue;
+            position:relative;
+            bottom:0;
```

Comment:
The tabular dataset on Crime, Healthcare and Property tabs was hidden by the Footer. In the _Layout.cshtml file, I have changed the CSS Code to increase the bottom padding of the body
and height of the footer by 60px. This enhances the User Experience, when they scroll to the tabs having tabular data they can now view the entire table on scrolling down the page.


Sources
https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/#:~:text=A%20footer%20is%20the%20last,is%20taller%20than%20the%20viewport.&text=If%20the%20content%20grows%20larger,want%20it%20to%20or%20not.

Reply:
Go one step further... move this to an external/linked css file.